mlir/test/Integration/Dialect/SparseTensor/python/tools/sparsifier.py

```diff
@@ -0,0 +1,39 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
I would expect a rename from  "sparse_compiler.py"
(or at least a removal of that file then?)

Reply:
I just pushed it with the file removed, so give it a second to check formattting.

Reply:
I don't see the rename/removal yet?